vmdb/app/services/orchestration_template_dialog_service.rb

```diff
@@ -0,0 +1,151 @@
+class OrchestrationTemplateDialogService
+  def create_dialog(dialog_label, template)
+    ActiveRecord::Base.transaction do
```

Comment:
This explicit transaction block is not needed if we change to build the dialog in memory first.